apps/backend/src/mcp/tools/device_management.py

```
@@ -360,43 +520,40 @@ async def get_device_info(
                 "monitoring_status": monitoring_status,
                 "query_info": {
                     "queried_identifier": device,
-                    "found_by": "uuid" if (device.count('-') == 4 and len(device) == 36) else "hostname_or_ip",
-                    "timestamp": datetime.now(timezone.utc).isoformat()
-                }
+                    "found_by": "uuid"
+                    if (device.count("-") == 4 and len(device) == 36)
+                    else "hostname_or_ip",
+                    "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Undefined name `timezone`

(F821)